src/index.d.ts

```diff
@@ -1,5 +1,6 @@
 import { CityJSONLoader } from './base/CityJSONLoader';
 import { CityJSONWorkerParser } from './parsers/CityJSONWorkerParser';
+import { CityObjectParser } from './parsers/CityObjectParser';
```

Comment:
`CityObjectParser` is an old way of parsing things which should be deprecated, in my opinion. The library took another direction since the introduction of `CityJSONWorkerParser` and with the use of custom shaders.

Reply:
`CityObjectParser` has been removed